resources/analysisTools/rnaseqworkflow/HIPO2_rnaseq_processing.sh

```diff
@@ -262,11 +262,20 @@ if [ "$RUN_ARRIBA" == true ]
 then
 	make_directory $ARRIBA_DIR
 	cd $ARRIBA_DIR
-	echo_run "$ARRIBA_READTHROUGH_BINARY -g $GENE_MODELS -i $ALIGNMENT_DIR/$STAR_SORTED_MKDUP_BAM -o ${SAMPLE}_${pid}_merged_read_through.bam"
-	echo_run "$ARRIBA_BINARY -c $ALIGNMENT_DIR/$STAR_CHIMERA_MKDUP_BAM -r ${SAMPLE}_${pid}_merged_read_through.bam -x $ALIGNMENT_DIR/$STAR_SORTED_MKDUP_BAM -a $GENOME_FA -k $ARRIBA_KNOWN_FUSIONS -g $GENE_MODELS -b $ARRIBA_BLACKLIST -o ${SAMPLE}_${pid}.fusions.txt -O ${SAMPLE}_${pid}.discarded_fusions.txt "
-	if [[ -f "${SAMPLE}_${pid}.fusions.txt" ]]
+
+	arribaCommand="$ARRIBA_BINARY -c '$ALIGNMENT_DIR/$STAR_CHIMERA_MKDUP_BAM' -x '$ALIGNMENT_DIR/$STAR_SORTED_MKDUP_BAM' -a '$GENOME_FA' -k '$ARRIBA_KNOWN_FUSIONS' -g '$GENE_MODELS' -b '$ARRIBA_BLACKLIST' -T -P -o '${SAMPLE}_$pid.fusions.txt' -O '${SAMPLE}_$pid.discarded_fusions.txt'"
```

Comment:
How about adding `gzip -9 '${SAMPLE}_$pid.discarded_fusions.txt'` after the call to Arriba? It's a big plain-text file that most users do not look into anyway. I usually compress it to save some disk space.

Reply:
Yes, I think this is a good suggestion. I'm going to add the call.